application-flashmessages-ui/src/main/resources/Flash/FlashSheet.xml

```diff
@@ -257,28 +257,34 @@ function summary() {
   var repeatFrequency = document.getElementById('Flash.FlashClass_0_repeatFrequency').value;
   var repeatDays      = [].map.call(document.querySelectorAll('input[name="Flash.FlashClass_0_repeatDays"]:checked'), function(day) { return day.value });
 
-  new Ajax.Request
-  (
-    '$xwiki.getURL("Flash.RepeatSummaryService", "view")',
-    {
-      method: 'get',
-      parameters:
+  //When one of the required values is not selected, repeat summary will not be displayed.
+  if (repeatInterval &amp;&amp; repeatFrequency){
+    document.getElementById("repeatSummary").parentElement.parentElement.classList.remove('hidden');
```

Comment:
You can define `document.getElementById("repeatSummary").parentElement.parentElement.classList` un a local variable. Also, you might want to use `'` to wrap `repeatSummary`.